alerting/src/main/kotlin/org/opensearch/alerting/actionv2/ExecuteMonitorV2Request.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.actionv2
+
+import org.opensearch.action.ActionRequest
+import org.opensearch.action.ActionRequestValidationException
+import org.opensearch.action.ValidateActions
+import org.opensearch.alerting.modelv2.MonitorV2
+import org.opensearch.common.unit.TimeValue
+import org.opensearch.core.common.io.stream.StreamInput
+import org.opensearch.core.common.io.stream.StreamOutput
+import java.io.IOException
+
+class ExecuteMonitorV2Request : ActionRequest {
+    val dryrun: Boolean
+    val manual: Boolean
+    val monitorV2Id: String? // exactly one of monitorId or monitor must be non-null
+    val monitorV2: MonitorV2?
+    val requestStart: TimeValue?
+    val requestEnd: TimeValue
+
+    constructor(
+        dryrun: Boolean,
+        manual: Boolean, // if execute was called by user or by scheduled job
+        monitorV2Id: String?,
+        monitorV2: MonitorV2?,
+        requestStart: TimeValue? = null,
```

Comment:
I went through the code, this is not actually used. It is only used once to compare this with requestEnd and if they are the same, it throws an exception. Based on that, we should just remove requestStart. This was needed in v1 monitors because query level and bucket level monitors lets you run the monitor execution within a specified time window. That does not happen for ppl monitors.